queries/public/playerStats.sql

```diff
@@ -0,0 +1,133 @@
+WITH avg_player_stats AS (
```

Comment:
By the way, you should probably name this file something other than player stats.  Either `historicalPlayerStatTotals.sql` or `seasonBySeasonPlayerStats.sql` or something like that.  Player stats is far too generic for people to know what this is supposed to show.

Also, I think you need to make a corresponding markdown file (the ones that are like `name.md`) for this query in order to determine what shows up on the public datasets website.

Reply:
The markdown file is optional, but it can be used to add context about what the data is supposed to be, which I would recommend on this one